saml-core/src/main/java/org/keycloak/saml/processing/core/saml/v2/util/AssertionUtil.java

```diff
@@ -557,21 +554,25 @@ public static AssertionType getAssertion(ResponseType responseType, PrivateKey p
         return responseType.getAssertions().get(0).getAssertion();
     }
 
-    public static boolean isAssertionEncrypted(ResponseType responseType) throws ProcessingException {
+    public static boolean isAssertionsEncrypted(ResponseType responseType) throws ProcessingException {
```

Comment:
Currently Keycloak only uses first assertion and this PR does not change this behaviour. Why to introduce this change?